### DISCUSSION TITLE
There is a v2.5.22 tag but no associated release ?

### DISCUSSION THREAD

**Discussion by emirot** [argoproj/argo-cd#15234] [General]
Hello,

I've seen that there is a tag https://github.com/argoproj/argo-cd/releases/tag/v2.5.22 but not 2.5.22 release.

the latest release for 2.5  is https://github.com/argoproj/argo-cd/releases/tag/v2.5.21 

Why is that ?

**Comment by crenshaw-dev** [comment]
Good catch! It's because I started the release, the release process was broken, and I never went and cleaned it up. 

Fixed the CI, I'll probably go back and cut that release. Fun tidbit for Argo trivia night.

**Reply by crenshaw-dev** [reply]
The @emirot release! https://github.com/argoproj/argo-cd/releases/tag/v2.5.22

**Reply by emirot** [reply]
Thank you so much @crenshaw-dev, oh looks like it's gonna be the last release for 2.5.
I have another question, would you recommend managing ArgoCD (by itself ) via an ArgoCD application ? Right now, I m relying on helm to install and upgrade.
